Sources/Controllers/TargetMenu/POI/OAPOIViewController.mm

```diff
@@ -157,19 +158,42 @@ - (NSAttributedString *) getAdditionalInfoStr
         NSMutableAttributedString *str = [[NSMutableAttributedString alloc] init];
         UIColor *colorOpen = UIColorFromRGB(color_ctx_menu_amenity_opened_text);
         UIColor *colorClosed = UIColorFromRGB(color_ctx_menu_amenity_closed_text);
-        for (auto info : _openingHoursInfo)
+        for (int i = 0; i < _openingHoursInfo.size(); i ++)
         {
+            auto info = _openingHoursInfo[i];
+
             if (str.length > 0)
                 [str appendAttributedString:[[NSAttributedString alloc] initWithString:@"\n"]];
-            
-            NSMutableAttributedString *s = [[NSMutableAttributedString alloc] initWithString:[NSString stringWithFormat:@"  %@", [NSString stringWithUTF8String:info->getInfo().c_str()]]];
+
+            NSString *time = [NSString stringWithUTF8String:info->getInfo().c_str()];
+
+            NSRegularExpression *regularExpressionAMPM = [NSRegularExpression regularExpressionWithPattern:@"\\d{1,2}:\\d{1,2} (A|P)M"
+                                                                                                   options:0
+                                                                                                     error:nil];
+            NSTextCheckingResult *matchAMPM = [regularExpressionAMPM firstMatchInString:time
+                                                                                options:0
+                                                                                  range:NSMakeRange(0, [time length])];
+            if (matchAMPM)
+            {
+                QLocale locale = QLocale();
+                time = [time stringByReplacingOccurrencesOfString:@"AM" withString:locale.amText().toNSString()];
+                time = [time stringByReplacingOccurrencesOfString:@"PM" withString:locale.pmText().toNSString()];
+            }
+
+            NSMutableAttributedString *s = [[NSMutableAttributedString alloc] initWithString:[NSString stringWithFormat:@"  %@", time]];
             NSTextAttachment *attachment = [[NSTextAttachment alloc] init];
-            attachment.image = [OAUtilities tintImageWithColor:[UIImage imageNamed:@"ic_travel_time"] color:info->opened ? colorOpen : colorClosed];
+            BOOL opened = info->fallback && i > 0 ? _openingHoursInfo[i - 1]->opened : info->opened;
+            attachment.image = [OAUtilities tintImageWithColor:[UIImage imageNamed:@"ic_travel_time"]
+                                                                             color:opened ? colorOpen : colorClosed];
             
             NSAttributedString *strWithImage = [NSAttributedString attributedStringWithAttachment:attachment];
             [s replaceCharactersInRange:NSMakeRange(0, 1) withAttributedString:strWithImage];
-            [s addAttribute:NSBaselineOffsetAttributeName value:[NSNumber numberWithFloat:-2.0] range:NSMakeRange(0, 1)];
```

Comment:
Do not do this, please. We have discussed pointless code changes multiple times!